tensorrt_llm/_torch/distributed/ops.py

```
@@ -959,3 +1001,93 @@ def forward(
+                )
+        except Exception as e:
+            logger.error(f"FlashInferVLLMAllReduce failed: {e}")
+            raise
+
+        return output_tensor
+
+    def forward(
+        self,
+        input: torch.Tensor,
+        *,
+        all_reduce_params: Optional[AllReduceParams] = None,
```

Ruff (0.14.14)

[warning] Unused method argument: `all_reduce_params`

(ARG002)